_posts/languages/ruby/2000-01-01-start.md

```diff
@@ -77,16 +77,10 @@ $ git push scalingo master
 If your need to install a custom version of Ruby, you can achieve that by
 specifying it in your Gemfile:
 
-### Ruby MRI 2.5.5
+### Ruby MRI 2.7.3
```

Comment:
Did you update our internal documentation explaining how to update the Ruby supported versions to specify to update this doc page?

Reply:
I'm doing it now!